lib/Selenium/Remote/Driver.pm

```diff
@@ -1008,6 +1008,46 @@ sub move_to {
     return shift->mouse_move_to_location(@_);
 }
 
+=head2 move_action
+
+ Description:
+    Move the mouse similarly to the mouse_move_to_location. But uses the
+    new webdrive actions API which is supported by geckodriver.
```

Comment:
mis-spelling: should be webdriver, not webdrive

Reply:
done

Reply:
Hi to the new maintainer btw!